website/docs/index.html.markdown

```diff
@@ -89,6 +89,7 @@ The following arguments are supported:
 * `init_helm_home` - (Optional) Initialize Helm home directory configured by the `home` attribute if it is not already initialized, defaults to true.
 * `install_tiller` - (Optional) Install Tiller if it is not already installed. Defaults to `true`.
 * `tiller_image` - (Optional) Tiller image to install. Defaults to `gcr.io/kubernetes-helm/tiller:v2.15.1`.
+* `connection_timeout` - (Optional) Number of seconds Helm will wait to establish a connection to tiller. Defaults to `5`.
```

Comment:
This should be 60. It's inconsistent with the actual default [here](https://github.com/terraform-providers/terraform-provider-helm/pull/374/files#diff-9f0ddb4b472596921b8dc5a236a017b2R81)